src/net.cpp

```diff
@@ -26,7 +26,6 @@
 #include <masternode/masternode-meta.h>
 #include <masternode/masternode-sync.h>
 #include <coinjoin/coinjoin.h>
-#include <evo/deterministicmns.h>
```

Comment:
We don't "need" it since it must be included in another header, but agree. dropped this commit. It's weird how removing this include "fixes" the linter even though it's clearly still being included from somewhere